src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -122,17 +135,16 @@ public void setPhones(List<Phone> phones) {
                     list.stream().map(name -> name.value).collect(Collectors.toList()));
         }
 
-        public void setEmails(List<Email> emails) {
+        public void setEmails(List<String> emails) {
             this.emails = emails;
         }
 
-        public Optional<List<Email>> getEmails() {
+        public Optional<List<String>> getEmails() {
             return Optional.ofNullable(emails);
         }
 
         public Optional<List<String>> getStringEmails() {
-            return getEmails().map(list ->
-                    list.stream().map(name -> name.value).collect(Collectors.toList()));
+            return getEmails();
```

Comment:
Not sure if this will break the functionality of other commands. WIll test it out after merging.